configuration/actions.md

```diff
@@ -396,56 +397,10 @@ Ephemeris supports translation of holidays into many languages. Localization sup
 1. [Portuguese](https://github.com/focus-shift/jollyday/tree/main/jollyday-core/src/main/resources/descriptions/holiday_descriptions_pt.properties)
 1. [Swedish](https://github.com/focus-shift/jollyday/tree/main/jollyday-core/src/main/resources/descriptions/holiday_descriptions_sv.properties)
 
-Feel free to extent this list by providing additional language support files.
-
-To enable localization,
-
-- copy the file for your language to your OH setup.
-  - again a folder in `$OH_CONF` folder, such as `$OH_CONF/services` is proposed.
-- use function 'Ephemeris.getHolidayDescription' to convert the name according to your localization file.
-
-## Installable Actions
```

Comment:
FYI this section was removed by the build script during build, so we can instead remove it from the source file and don't need to adjust the build script.